clang/include/clang/Lex/PPCallbacks.h

```diff
@@ -127,8 +127,12 @@ class PPCallbacks {
   /// \param RelativePath The path relative to SearchPath, at which the include
   /// file was found. This is equal to FileName except for framework includes.
   ///
-  /// \param Imported The module, whenever an inclusion directive was
-  /// automatically turned into a module import or null otherwise.
+  /// \param SuggestedModule The module, whenever an inclusion directive was
+  /// considered to be automatically turned into a module import, or null
```

Comment:
I think this comment should be updated since it's no longer always a "module import".

Reply:
I'm not sure I understand. The documentation says this is the module that was **considered** for importing (not that it actually got imported). Should I clarify somehow?

Reply:
"considered to be" is ambiguous:

Reply:
"was considered to be" most commonly it means you are describing the result of a consideration -- "Clang is considered to be a good compiler" -- rather than simply saying that something was considered but without reflecting the final judgment.  It's ambiguous here.

Suggestion: "The module suggested for this header, if any. See \p ModuleImported for whether this include was translated into a module imported"